### DISCUSSION TITLE
TFT 24 v1.1 update x.26 (20210217) black screen (Closed)

### DISCUSSION THREAD

**Discussion by Pirat-1989** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1729] [Q&A]
I can't update the screen firmware. The firmware passes, followed by a black screen. Neither the icons nor the font are stitched. What could be the problem? I attach a video with what is happening.
![2021-03-14_17-10-06](https://user-images.githubusercontent.com/80626953/111067902-34ea9500-84e8-11eb-9d80-2563136a6381.png)
[https://youtu.be/_y6aVA6-Ne4](url)

**Comment by oldman4U** [comment]
Try a new version after PR #1703.

**Reply by Pirat-1989** [reply]
Thank you, it seems that I reviewed everything, but I did not see that the firmware was updated not so long ago.

**Comment by oldman4U** [comment]
Or this could help: https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/issues/1684

**Comment by oldman4U** [comment]
There is almost every day a new firmware available. A new version will be
available soon.

Does it work?

Pirat-1989 ***@***.***> schrieb am Di. 16. März 2021 um 18:19:
